packages/fx-core/src/plugins/resource/frontend/utils.ts

```diff
@@ -66,6 +71,11 @@ export class Utils {
     return new Promise((resolve, reject) => {
       Logger.info(`Start to run command: "${command}".`);
 
+      // Drive letter should be uppercase, otherwise when we run webpack in exec, it fails to resolve nested dependencies.
+      if (os.platform() === "win32") {
+        workingDir = this.capitalizeFirstLetter(path.resolve(workingDir ?? Constants.EmptyString));
```

Comment:
`path.resolve("")` return current working directory.

Reply:
You want to regard it as an error? What cwd do we want to pass to exec if we give it ""?

Reply:
If we pass "" or undefined, then I assume the command can be run in any dir. It should be ok to use cwd.
Do you worry we may get empty path from vscode?

Reply:
No, I just wonder the cwd of exec if the path is relative.
If path is relative and the cwd of subprocess starts with lower-case drive letter, then we'd better also upper case the drive letter for it.
For me, if cwd is `""`, then it means `"."`. I think exec also regard `""` as `"."`. (need verify)

Reply:
exec indeed treats `""` as `"."`. 
As for the drive letter, it depends on the exact path starts with lowercase or uppercase. 
For example, if I start a PowerShell in `c:\` and run exec(""), its path is `c:\`.
Anyway `path.resolve` always return absolute path, I think we don't need to worry about this.

Reply:
good.